pkg/manager/member/dm_master_member_manager.go

```diff
@@ -585,10 +588,11 @@ func getNewMasterSetForDMCluster(dc *v1alpha1.DMCluster, cm *corev1.ConfigMap) (
 		},
 	}
 
-	masterLabel := label.NewDM().Instance(instanceName).DMMaster()
 	setName := controller.DMMasterMemberName(dcName)
-	podAnnotations := CombineAnnotations(controller.AnnProm(8261), baseMasterSpec.Annotations())
+	stsLabels := label.NewDM().Instance(instanceName).DMMaster()
+	podLabels := util.CombineStringMap(stsLabels, baseMasterSpec.Labels())
 	stsAnnotations := getStsAnnotations(dc.Annotations, label.DMMasterLabelVal)
+	podAnnotations := util.CombineStringMap(stsAnnotations, controller.AnnProm(8261), baseMasterSpec.Annotations())
```

Comment:
Currently, the stsAnnotations returned in the last line only include the delete slot annotation, which is used for advanced statefulset, no need to add it to the Pod annotations.